Mogakco/Sources/Data/DataSources/Remote/ChatDataSource.swift

```diff
@@ -9,38 +9,86 @@
 import RxSwift
 import Firebase
 
-struct ChatDataSource: ChatDataSourceProtocol {
+final class ChatDataSource: ChatDataSourceProtocol {
+    
+    var listener: ListenerRegistration?
+    var page: DocumentSnapshot!
```

Comment:
```suggestion
    var listener: ListenerRegistration?
    var page: DocumentSnapshot?
```
강제 언래핑은 지양하는게 좋을 것 같습니다!